Automattic/wp-calypso
# Update list of features in Anti-spam card

### Changes proposed in this Pull Request

This PR is a follow up of #49277. It adds the "Advanced stats" bullet point  back, and removes the "Priority support" one.

### Testing instructions

- Run the PR in Jetpack cloud or Calypso
- Visit the pricing/plans page
- Check the copy of the Anti-spam card against the capture below, for both monthly and yearly terms

### Screenshots

#### Before
<img width="362" alt="Screen Shot 2021-01-27 at 2 23 23 PM" src="https://user-images.githubusercontent.com/1620183/106042905-aa6ee300-60ab-11eb-8383-cfef3085b1a3.png">

#### After
<img width="360" alt="Screen Shot 2021-01-27 at 2 24 53 PM" src="https://user-images.githubusercontent.com/1620183/106042894-a80c8900-60ab-11eb-8bd8-e01d5a80679b.png">
